testsuite/CHANGELOG

```diff
@@ -14,8 +14,12 @@ and https://github.com/MDAnalysis/mdanalysis/wiki/UnitTests
 
 ------------------------------------------------------------------------------
 ??/??/16 jbarnoud, orbeckst, fiona-naughton, manuel.nuno.melo, richardjgowers
+         tyler.je.reddy
 
   * 0.15.1
+    - added test for LeafletFinder handling a selection string
```

Comment:
Can you please just update 0.15.1 to 0.16.0? I forgot to do it in an earlier PR and it seems not worth a fresh commit...